src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,284 @@
+<template>
+  <div>
+    <div>
+      <label>{{ selectLabel }} <span v-if=isRequired aria-label='Required'>*</span></label>
+    </div>
+    <div 
+      :aria-expanded="opened"
+      :aria-label="selectLabel"
+      :aria-owns="id"
+      :aria-required="isRequired"
+      :id="'dropdown' + id"
+      @keyup.32.prevent.stop="toggleList"
+      role="combobox"
+      tabindex="0"
+      v-model="selectLabel"
+    >
+      <ul
+        :class="ulClass"
+        :id="id"
+        role="listbox"
+      >
+        <li 
+          :class="dropdownClass" 
+          @click="toggleList"
+          >{{ selectedOption ? selectedOption.text : placeholder }}
+          <span class="dropdown-arrow"><i class="fa fa-caret-down" aria-hidden="true"></i></span>
+        </li>
+        <span v-show="selectErrors.has('selectedOption')" class="error" data-vv-as="selectLabel">{{ selectErrors.first('selectedOption') }}</span>
```

Comment:
Why is the error span within the list options? The span should have an id and then the combobox div should have aria-describedby which references that id.

Reply:
its not in the list options, its under the dropdown?  I'll add the id and describedby

Reply:
It's a child of the `ul` element rather than being after it. That seems like the wrong place for it.